refresh/templates/common/README.scaffold.md

```diff
@@ -66,16 +66,21 @@ To build and run the application:
 
 {{#if docker}}
 #### Docker
-A description of the files related to Docker can be found in the [Docker files](#docker-files) setion. To build the two docker images, run the following commands from the root directory of the project:
-* `docker build -t myapp-run .`
-* `docker build -t myapp-build -f Dockerfile-tools .`
+A description of the files related to Docker can be found in the [Docker files](#docker-files) section. To build the two Docker images, run the following commands from the root directory of the project.
+
+First, build the 'tools' image. This image contains a full Linux Swift toolchain and is capable of compiling your application for Linux:
+* `docker build --tag myapp-build --file Dockerfile-tools .`
 You may customize the names of these images by specifying a different value after the `-t` option.
 
-To compile the application using the tools docker image, run:
-* `docker run -v $PWD:/swift-project -w /swift-project myapp-build /swift-utils/tools-utils.sh build release`
+To compile the application for Linux using the tools Docker image, run:
+* `docker run --volume $PWD:/swift-project --workdir /swift-project myapp-build /swift-utils/tools-utils.sh build release`
+This produces a `.build-ubuntu` directory which will be incorporated into your final image.
+
+Now that your application has been built, you can build the final run image:
+* `docker build --tag myapp-run .`
 
-To run the application:
-* `docker run -it -p 8080:8080 -v $PWD:/swift-project -w /swift-project myapp-run sh -c .build-ubuntu/release/{{executableName}}`
+Finally, to run the application:
+* `docker run -it --publish 8080:8080 myapp-run`
```

Comment:
The `Dockerfile` already contains a `COPY` of pwd as `/swift-project`, so mapping a volume is unnecessary.  It also contains a `CMD` to cd to /swift-project and run the executable, so that part is also superfluous.

Reply:
(this is now consistent with https://www.kitura.io/docs/deploying/docker.html - we should make the same changes to expand the flags as I agree, it reads much nicer)